src/rdds/lib/hdf5/viewer.py

```diff
@@ -0,0 +1,61 @@
+from h5py import File, Group, Dataset
+from typing import List, Union
+import numpy as np
```

Comment:
```suggestion
import numpy
```
Keep the namespace clean and explicit 😄

Reply:
I'm trying to stick to the numpy convention described here: https://numpy.org/doc/stable/user/absolute_beginners.html#how-to-import-numpy